packages/cli/yaml/yaml-schema/src/ast/visitors/services/visitHttpService.ts

```diff
@@ -269,7 +269,8 @@ async function visitEndpoint({
                     example
                 });
             }
-        }
+        },
+        pagination: noop
```

Comment:
@amckinney do we want to add a `fern check` rule to verify that all the parameters exist?

Reply:
Discussed offline, we'll follow-up with the `fern check` and fern compiler changes in a follow-up PR. This at least gets the interface into IRv35 so the generators can start to act upon the shape.